docs/docsite/rst/userguide/rbac.rst

```diff
@@ -0,0 +1,513 @@
+.. _rbac-ug:
+
+Role-Based Access Controls
+==========================
+
+.. index::
+   single: role-based access controls
+   pair: security; RBAC
+
+Role-Based Access Controls (RBAC) are built into AWX and allow administrators to delegate access to server inventories, organizations, and more. Administrators can also centralize the management of various credentials, allowing end users to leverage a needed secret without ever exposing that secret to the end user. RBAC controls allow AWX to help you increase security and streamline management.
+
+This chapter has two parts: the latest RBAC model (:ref:`rbac-dab-ug`) and the :ref:`existing RBAC <rbac-legacy-ug>` implementation.
+
+.. _rbac-dab-ug:
+
+DAB RBAC
+---------
+
+.. index::
+   single: DAB
+   single: roles   
+   pair: DAB; RBAC
+
+This section describes the latest changes to RBAC, involving use of the ``django-ansible-base`` (DAB) library, to enhance existing roles, provide a uniformed model that is compatible with platform (enterprise) components, and allow creation of custom roles. However, the internals of the system in the backend have changes implemented, but they are not reflected yet in the AWX UI. The change to the backend maintains a compatibility layer so the “old” roles in the API still exists temporarily, until a fully-functional compatible UI replaces the existing roles. 
+
+New functionality, specifically custom roles, are possible through direct API clients or the API browser, but the presentation in the AWX UI might not reflect the changes made in the API.
+
+The new DAB version of RBAC allows creation of custom roles which can be done via the ``/api/v2/role_definitions/`` endpoint. Then these can only be assigned using the new endpoints, ``/api/v2/role_user_assignments/`` and ``/api/v2/role_team_assignments/``.
+
+If you do not want to allow custom roles, you can change the setting ``ANSIBLE_BASE_ALLOW_CUSTOM_ROLES`` to ``False``. This is still a file-based setting for now.
+
+New “add” permissions are a major highlight of this change. You could create a custom organization role that allows users to create all (or some) types of resources, and apply it to a particular organization. So instead of allowing a user to edit all projects, they can create a new project, and after creating it, they will automatically get admin role just for the objects they created.
+
+
+Resource access for teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section provides a reference for managing team roles within individual resources as shown in the new UI and the corresponding API calls.
+
+Access the resource's **Team Access** tab to manage the team roles.
+
+.. image:: ../common/images/rbac_jt_team_access.png
+
+To obtain a list of team role assignments from the API: 
+
+::
+
+   GET <awxAPI>/role_team_assignments/?object_id=<template_id>&content_type__model=jobtemplate
```

Comment:
Can we replace all the references to `<awxAPI>/` with `https://<awx-host>/api/v2/` or `/api/v2/`?

Reply:
Yes, for sure!